pkg/server/plugin/upstreamca/disk/disk.go

```diff
@@ -72,7 +58,7 @@ func (m *diskPlugin) Configure(ctx context.Context, req *spi.ConfigureRequest) (
 		return nil, fmt.Errorf("unable to read %s: %s", config.CertFilePath, err)
 	}
 
-	block, rest = pem.Decode(certPEM)
+	block, rest := pem.Decode(certPEM)
```

Comment:
(optional) while we're in here, we may consider: 1) using pemutil.LoadCertificate, 2) adding a validation step that the loaded private key matches the key in the certificate.